File: AppliedBarberShop/app/src/main/java/edu/ucne/appliedbarbershop/data/local/models/Servicio.kt
package edu.ucne.appliedbarbershop.data.local.models

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Servicios")
data class Servicio(
    @PrimaryKey(autoGenerate = true)
    var servicioId: Int,
    val nombre: String,
    val imagen: String?,
    val usuarioCreacionId: Int,
    val usuarioModificacionId: Int?,
    val status: Int
)

File: ApiBarberShop/ApiBarberShop/Models/Servicio.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ApiBarberShop.Models;

public partial class Servicio
{
    [Key]
    public int ServicioId { get; set; }

    public string Nombre { get; set; } = null!;

    public string? Imagen { get; set; }

    public DateTime FechaCreacion { get; set; }

    public DateTime? FechaModificacion { get; set; }

    public int UsuarioCreacionId { get; set; }

    public int? UsuarioModificacionId { get; set; }

    public int Status { get; set; }

    //public virtual ICollection<Cita>? Cita { get; } = new List<Cita>();
}
